pkg/chartverifier/reportBuilder.go

```diff
@@ -64,8 +64,9 @@ func (r *reportBuilder) SetToolVersion(version string) ReportBuilder {
 	return r
 }
 
-func (r *reportBuilder) SetProfile(profileName string) ReportBuilder {
-	r.Report.Metadata.ToolMetadata.Profile = profileName
+func (r *reportBuilder) SetProfile(vendorType profiles.VendorType, version string) ReportBuilder {
+	r.Report.Metadata.ToolMetadata.Profile.VendorType = string(vendorType)
```

Comment:
I think `string()` is redundant here since `profiles.VendorType` is already `string`

Reply:
If i remove string it complains: Cannot use 'vendorType' (type profiles.VendorType) as type string.